packages/react/src/providers/FuelProvider.tsx

```diff
@@ -10,6 +11,7 @@ export { useConnectUI } from './FuelUIProvider';
 
 type FuelProviderProps = {
   ui?: boolean;
+  showWebWallet?: boolean;
```

Comment:
can we invert it to `hideWebWallet` ? then the wallet will be available by default